docs/the-new-architecture/why.md

```diff
@@ -3,5 +3,40 @@ id: why
 title: Why A New Architecture
 ---
 
-This section briefly explains why we decided to move away from the old architecture, and it describes succinctly the main benefits of adopting the new architecture.
-It points to the [Architecture](https://reactnative.dev/architecture/overview) tab of the website for a deep dive into the ‘why’s.
+The goal of the New Architecture is to solve some of the issues that afflicted the Old Architecture in terms of performance and flexibility. This section provides the basic context to understand the Old Architecture limitations and how it has been possible to overcome them with the New Architecture.
+
+This is not a technical deep dive: for further technical information, refer to the [Architecture](https://reactnative.dev/architecture/overview) tab of the website.
+
+## Old Architecture's Issues
+
+The Old Architecture used to work by serializing all the data that has to be passed from the JS layer to the native layer using a component called `The Bridge`. `The Bridge` can be imagined as a bus where the producer layer sent some data for the consumer layer. The consumer could read the data, deserialize it and execute the required operations.
```

Comment:
```suggestion
The Old Architecture used to work by serializing all the data that has to be passed from the JS layer to the native layer using a component called _The Bridge_. The Bridge can be imagined as a bus where the producer layer sent some data for the consumer layer. The consumer could read the data, deserialize it and execute the required operations.
```

Reply:
Addressing here but valid for the rest of the doc. Backtick should be used only for code or commands (i.e. anything that needs to be typed by a user). Here italic is probably better. Similarly for the rest fo the document.